python_multipart/multipart.py

```diff
@@ -1102,10 +1102,17 @@ def data_callback(name: CallbackName, end_i: int, remaining: bool = False) -> No
             c = data[i]
 
             if state == MultipartState.START:
+                # Stop parsing if there is no boundary within the first chunk
+                if i == 16:
```

Comment:
Why 16?

Reply:
Yea, I'm not happy about that either. Two line breaks should be allowed, that's not that uncommon. More are uncommon, but who knows? It's just an arbitrary number between 2 and "too much to waste time on it". Maybe ignore the second commit for now and just remove the log lines?

Reply:
The comment is lying, i was first checking for `i == length-1` but that broke tests. This needs more work I guess.